client/my-sites/checkout/checkout/checkout-system-decider.js

```diff
@@ -91,6 +92,7 @@ export default function CheckoutSystemDecider( {
 			couponCode={ couponCode }
 			isComingFromSignup={ isComingFromSignup }
 			isComingFromGutenboarding={ isComingFromGutenboarding }
+			isGutenboardingCreate={ isGutenboardingCreate }
```

Comment:
Do we need both of these? (`isComingFromGutenboarding` and `isGutenboardingCreate`?) How are they different?

Reply:
Maybe we can work a bit on the naming. One is for the launch flow and the other for current flow (domain + plan purchase for existing site). We have 2 main reasons to have them both:
1. Functional (immediate need): we need this [conditional](https://github.com/Automattic/wp-calypso/pull/40813/files#diff-29f2e23b8d6785007185724253016b67R60) because `launch` step needs to be skipped when going back from Checkout to `plans-launch` in `frankenflow`.
2. Styling (potential need): to have different styling for create/launch checkout pages.